connector-common-http/src/main/java/com/redhat/cloud/notifications/connector/http/HttpExceptionProcessor.java

```diff
@@ -38,7 +39,12 @@ public class HttpExceptionProcessor extends ExceptionProcessor {
     protected void process(Throwable t, Exchange exchange) {
         if (t instanceof HttpOperationFailedException e) {
             exchange.setProperty(HTTP_STATUS_CODE, e.getStatusCode());
-            if (e.getStatusCode() >= 400 && e.getStatusCode() < 500 && e.getStatusCode() != SC_TOO_MANY_REQUESTS) {
+            if (e.getStatusCode() >= 300 && e.getStatusCode() < 400) {
```

Comment:
If the entire 3xx family about redirections?

Reply:
Yes according https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/300